### DISCUSSION TITLE
云服务的推广可以加入一个关闭按钮吗？现在一直在上面闪，比较难受

### DISCUSSION THREAD

**Discussion by queyangdeyangxiansheng** [lobehub/lobehub#3221] [General Question] [answered]
云服务的推广可以加入一个关闭按钮吗？现在一直在上面闪，比较难受

**Comment by arvinxx** [comment, marked as answer]
这个不会加。如果不想看到的话，有两个做法：

1. 自己部署一个 LobeChat 实例（Vercel 或者 Docker 都可以）
2. 使用 Cloud 版本

Preview 的定位是LobeChat 私有化部署版本的 Demo ，不建议在日常中使用。
